index.html

```diff
@@ -335,19 +336,19 @@ <h2>
           <ol class="algorithm">
             <li>Let |promise:Promise| be [=a new promise=].
             </li>
-            <li>When a window is in focus, if the [=this=]'s [=shadow-including
-            root=] is the [=navigable/active document=] of a [=Document/browsing
-            context=] (or has an [=tree/ancestor=] [=browsing context=]) which is not in
-            focus:
+            <li>When a {{Window/window}} is in [=Window/focus=], if the [=this=]'s
+            [=shadow-including root=] is the [=navigable/active document=] of a
+            [=Document/browsing context=] (or has an [=tree/ancestor=]
+            [=browsing context=]) which is not in focus:
```

Comment:
```suggestion
            [=browsing context=]) that is not in focus:
```

Reply:
We definitely need some kind of test or example for this... this part is super confusing and I can't tell if this is important or not for security, privacy, or use annoyance reasons.

Reply:
Done, as for the comment to original algorithm I will propose an updated algorithm in https://github.com/w3c/pointerlock/pull/49/files#r1373662771 and review it again from there.